algorithms/tree_isomorphisms.py

```diff
@@ -58,7 +58,8 @@ def trees_count_isomorphisms(T1: "Graph", T2: "Graph", count_flag: "Bool"):
                 children_auto = {}
                 for n in v.neighbours:
                     if n.level > v.level:
-                        children_names.append(n.name)
+                        for _ in range(n.n_twins):
```

Comment:
Zorgt dit er ook voor dat je er niet vaker overheen hoeft te loopen?

Reply:
Wat bedoel je met 'dit'? 

Als je die underscore bedoelt, dat zorgt er voor dat ik zo vaak de naam toevoeg als dat er twins waren van die vertex.

Reply:
Ik bedoel dat je op dit moment nu meer namen toevoegt, zorgt dit er dan ook voor dat je deze loop minder vaak door hoeft? Of is het zo dat als je hier niet voor zoveel twins er zijn de namen toevoegt, dat je er dan te weinig hebt?

Reply:
Ja, want als je bijvoorbeeld twee vertices hebt waarvan de eerste 2 children heeft die leafs zijn en de ander heeft 3 children die leafs zijn. Dan zijn ze niet hetzelfde. Maar als je de twin removal doet, blijven ze uiteindelijk allebei met 1 leaf achter. Om te voorkomen dat ze naar elkaar gemapt kunnen worden moet je ze dus wel de naam geven alsof ze 2 en 3 kinderen hebben.